app/livechat/imports/server/rest/sms.js

```diff
@@ -45,9 +45,8 @@ const defineVisitor = (smsNumber) => {
 		});
 	}
 
```

Comment:
Can you please validate if the department exists before assigning it to the visitor? 👀

Reply:
Fixed this at line 74, that will ensure wrong department never reaches here